extensions/linz/README.md

```diff
@@ -22,12 +22,12 @@ extension which adds constraints to default STAC schema properties.
 - [JSON Schema](./schema.json)
 - [Changelog](./CHANGELOG.md)
 
-## Item Fields
+## Item Properties
 
-| Field Name           | Type                | Description                                                                                                                                                  |
-| -------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| linz:geospatial_type | string              | **REQUIRED**. A general description of the type of content that can be found in the dataset. See the [list of accepted geospatial types](#geospatial-types). |
-| processing:software  | Map<string, string> | Recommended. The software and versions which were used to generate the dataset. See [reference](https://github.com/stac-extensions/processing).              |
+| Property Name                   | Type                | Description                                                                                                                                     |
+| ------------------------------- | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| properties/linz:geospatial_type | string              | A general description of the type of content that can be found in the dataset. See the [list of accepted geospatial types](#geospatial-types).  |
```

Comment:
I think people tend to leave off the prefix if it's mentioned in the title where the metadata is?
https://github.com/stac-extensions/version#item-properties-and-asset-or-collection-fields and https://github.com/stac-extensions/file#asset--link-object-fields